src/Umbraco.Web/Scheduling/SchedulerComponent.cs

```diff
@@ -120,7 +125,7 @@ private IBackgroundTask RegisterScheduledPublishing()
         {
             // scheduled publishing/unpublishing
             // install on all, will only run on non-replica servers
-            var task = new ScheduledPublishing(_publishingRunner, 60000, 60000, _runtime, _contentService, _umbracoContextFactory, _logger);
+            var task = new ScheduledPublishing(_publishingRunner, DefaultDelayMilliseconds, OneMinuteMilliseconds, _runtime, _contentService, _umbracoContextFactory, _logger);
```

Comment:
Is this change from 60000 ms to 180000 ms intended?

Reply:
Yes, we start these background tasks far too quickly after startup, sometimes on cloud it takes a minute to even boot (depending on what folks are doing with their code) and we should initiate BG tasks before boot is fully complete. In seb's logs I saw calls for running through all of the healthchecks before the whole boot process was done which is insane, especially when you consider overlapping app domains and multiple restarts.